server/jobs/onboarding/freckleJob.js

```diff
@@ -0,0 +1,11 @@
+import { fetchNewPlacements } from '../../modules/allocations';
+import { createProject } from '../../modules/freckle/projects';
+
+export default () => {
```

Comment:
Hi @Hector101 
This function is not following the signature of how the job-functions should be created.

The logic of fetching data from allocaitons, looping through the fetched data and other related activites would be done somewhere else and not inside the job.

If you are yet to rebase from the `develop` branch, please do so that you can have the latest changes.

You can take a look at [this job](https://github.com/andela/bp-esa-backend/pull/51/files#diff-847bd18292dfb7dc549471baf8cdf151) for comparison